R/bfgs.R

```diff
@@ -0,0 +1,26 @@
+#'
+bfgs <- function(design, outcome, model = c("logit", "linear")) {
```

Comment:
you can avoid the model parameter (and all this if/else logic) entirely by making obj_fn and obj_grad arguments to the function.